docs/add-ons/presentation.md

```diff
@@ -43,27 +43,51 @@ If you are in need of a guide on keeping a changelog, we would recommend checkin
 
 ## AppArmor
 
-You can use own security profile for your add-on with AppArmor. By default it is enabled and uses the Docker default profile. Putting a `apparmor.txt` file into your add-on folder, will load that file as the primary profile instead. Use the config options to set the name of that profile.
+In the event an API call returns something you, as a developer were not expecting, access to too many resouces could be a liability for your users. As an add-on developer it is your responsibility to ensure your Add-on will not ruin your user's machine, or perform actions that you would never expect. That's where AppArmor comes in.  While your tallents in input validation, handling sensitive data and other defensive programming tactics are not being judged here, AppArmor is your Add-on's second line of defense against malicious API calls, malformed settings or other forms of system hijacking. 
+
+By default, AppArmor gives you a certain level of security by restricting some general actions that are deemed inappropriate for a Docker container. You can read more about Docker's AppArmor implementation on the [Docker Security page](https://docs.docker.com/engine/security/apparmor/). 
+
+As for the Home Assistant Operating System implementation, you can activate your own custom AppArmor profile by putting a `apparmor.txt` file into your add-on folder. Adding your own `apparmor.txt` will load that file as the primary AppArmor profile instead of the default implementation. On top of knowing your Add-on will run in a constrained and effective manner, writing your own custom `apparmor.txt` file will earn your Add-on a security point after your Add-on is installed, thus improving your user's confidence and perception of your Add-on.
+
+An apparmor.txt goes in the same folder as your config.json file. Below is an example `apparmor.txt`. Replace `ADDON_SLUG` with the slug defined in your config. 
 
 apparmor.txt
 ```txt
 #include <tunables/global>
 
 profile ADDON_SLUG flags=(attach_disconnected,mediate_deleted) {
   #include <abstractions/base>
+  
+  # Capabilities
+  capability,
+  file,
+
+  capability setgid,
+  capability setuid,
```

Comment:
While setting these in general Docker conventions, in our ecosystem the use is not advised. I would therefore recommend them to be removed from the default example.